example/android/gradle.properties

```diff
@@ -12,17 +12,13 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-#org.gradle.jvmargs=-Xmx4g
+org.gradle.jvmargs=-Xmx4g
 android.useAndroidX=true
 android.enableJetifier=true
-#android.jetifier.ignorelist=moshi-1.13.0
-#android.jetifier.blacklist=moshi-1.13.0
-android.jetifier.blacklist = bcprov-jdk15on
-org.gradle.jvmargs=-Xmx1536M --add-exports=java.base/sun.nio.ch=ALL-UNNAMED --add-opens=java.base/java.lang=ALL-UNNAMED --add-opens=java.base/java.lang.reflect=ALL-UNNAMED --add-opens=java.base/java.io=ALL-UNNAMED --add-exports=jdk.unsupported/sun.misc=ALL-UNNAMED
 ## Uncomment the properties below and set their correct value.
 
 ## The Application ID displayed on the Credentials tab in the Square Application Control Panel.
-## SQUARE_READER_SDK_APPLICATION_ID=
```

Comment:
Do we really need to remove the comment?

Reply:
We don't, it was unintentional change at start, I didn't revert it back since user still will uncomment it when he will try to build example app so I didn't revert it

Reply:
do you still want to keep them commented?

Reply:
I'd like to keep them commented for better error message when they forget to update this file. 

If it's commented, user will know `SQUARE_READER_SDK_APPLICATION_ID` is missing from the config. If it's not commented, they will see empty value error which is not explicit.

Reply:
Ah, I see. Yep it makes sense

Reply:
commented out keys